cg/services/post_processing/pacbio/data_transfer_service/data_transfer_service.py

```diff
@@ -0,0 +1,34 @@
+from cg.services.post_processing.abstract_classes import PostProcessingDataTransferService
+from cg.services.post_processing.pacbio.data_transfer_service.dto import (
+    PacBioDTOs,
+    PacBioSampleSequencingMetricsDTO,
+    PacBioSequencingRunDTO,
+    PacBioSMRTCellDTO,
+)
+from cg.services.post_processing.pacbio.data_transfer_service.utils import (
+    get_sample_sequencing_metrics_dtos,
+    get_sequencing_run_dto,
+    get_smrt_cell_dto,
+)
+from cg.services.post_processing.pacbio.metrics_parser.metrics_parser import PacBioMetricsParser
+from cg.services.post_processing.pacbio.metrics_parser.models import PacBioMetrics
+from cg.services.post_processing.pacbio.run_data_generator.run_data import PacBioRunData
+
+
+class PacBioDataTransferService(PostProcessingDataTransferService):
+
+    def __init__(self, metrics_service: PacBioMetricsParser):
+        super().__init__(metrics_service=metrics_service)
```

Comment:
Why are you calling the parent constructor like this? Seems wonky! 😄 
I'd expect the constructor to look like
```python
def __init__(self, metrics_service: PacBioMetricsParser):
    self.metrics_service = metrics_service
```

Reply:
I read some more and it might make sense to call the parent constructor, I'll leave it to you! I guess you want to enforce a strict design of the components.

I'm more used to interfaces which work a bit differently, only specifying the expected methods to be available on the class implementing the interface. A single class can also implement multiple interfaces

Reply:
We are fixing this later on.